.github/workflows/release-please.yml

```diff
@@ -6,10 +6,14 @@ name: release-please
 
 jobs:
   release-please:
+    # The secret HCLOUD_BOT_TOKEN is only available on the main repo, not in forks.
+    if: github.repository == 'hetznercloud/hcloud-cloud-controller-manager'
```

Comment:
Mm, we could also detect if the secret is set here, right? Not sure how hard we should work here to ensure releases can be cut for forks, though.

Reply:
> Not sure how hard we should work here to ensure releases can be cut for forks, though.

Not at all in my opinion, thats why I just deactived the workflow for forks.

If left running for forks, this will create notifications of failing workflows for people having forks, i encountered this issue with the cluster-api-provider-hetzner.